src/main/java/rx/schedulers/ExecutorScheduler.java

```diff
@@ -96,7 +93,10 @@ public Subscription schedule(Action0 action) {
         @Override
         public void run() {
             do {
-                queue.poll().run();
+                ScheduledAction sa = queue.poll();
+                if (!sa.isUnsubscribed()) {
```

Comment:
Well that's a sad bug we had sitting in there allowing unsubscribed tasks to still execute!